GitCommands/Settings/AppSettings.cs

```diff
@@ -1928,6 +1928,18 @@ public static int RepoObjectsTreeStashesIndex
             set => SetInt("RepoObjectsTree.StashesIndex", value);
         }
 
+        public static string PrioritizedBranchNames
+        {
+            get => GetString("PrioritizedBranchNames", "main.*|master.*|release/.*");
```

Comment:
Why are `main.*` and `master.*`?

Reply:
Regex for common most important branches.
Like master, master_v3 etc

Reply:
🤣 "master_v2_final"

Reply:
The intention is to have regex for integration branches where there is development. GE is not doing active development in other than master.
A different team could have developed in master_mono still cherry picking interesting changes.